src/IssueViz.Security/HotspotsList/TableDataSource/CustomColumns/PriorityTableColumnDefinition.cs

```diff
@@ -29,13 +29,12 @@ namespace SonarLint.VisualStudio.IssueVisualization.Security.HotspotsList.TableD
     internal class PriorityTableColumnDefinition : TableColumnDefinitionBase
     {
         public const string ColumnName = "ReviewPriority";
+        public const int Width = 120;
 
         public override string Name { get; } = ColumnName;
 
-        public override double DefaultWidth => 50;
+        public override string DisplayName { get; } = "Priority";
```

Comment:
Minor unrelated change: changed to single initialization as the value will never change.